api/classes/publisher.js

```diff
@@ -127,262 +120,276 @@ function deleteFileRemotely(file, root) {
   });
 }
 
-function modifyPublishedFiles(action, project, user, files, remixMetadata) {
-  return Promise.resolve()
-  .then(function() {
-    if (files.length) {
-      return files.mapThen(function(file) {
-        return action(file, '/' + user.get('name') + '/' + project.get('id'), remixMetadata);
-      });
+
+/**
+ * Record fetching helpers
+ */
+function fetchProjectModel(id) {
+  return Projects.query({
+    where: {
+      id: id
     }
-    return action(files, '/' + user.get('name') + '/' + project.get('id'), remixMetadata);
+  }).fetch();
+}
+
+function fetchUserForProject() {
+  var self = this;
+
+  return UserQueries.getOne(self.project.user_id)
+  .then(function(user) {
+    self.user = user;
+    return user;
```

Comment:
Actually, it is used in the `createOrUpdatePublishedProject` method and I left it this way since it seemed to make more semantic sense - "fetch" should actually fetch which sorta implies return